### PR TITLE
Updated request size limit

### PR DESCRIPTION
**Pull request by shsimeonova**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/375

Linked pull requests from another repos (if any): https://github.com/SoftUni-Internal/judge-workers-deployment/pull/35

Added .net configuration for request size limit

### CODE REVIEW

File: Services/Common/OJS.Services.Common.Models/Configurations/HttpConfig.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+namespace OJS.Services.Common.Models.Configurations;
+
+public class HttpConfig : BaseConfig
+{
+    public override string SectionName => "Http";
+
+    public int MaxRequestSizeLimit { get; set; } = 35000000;
```
Comment (gogo4ds) [inline Services/Common/OJS.Services.Common.Models/Configurations/HttpConfig.cs:7]: Can you change the value to 35_000_000 and add comment for easy reading e.g. `= 35_000_000 // 35MB`

File: Servers/Infrastructure/OJS.Servers.Infrastructure/Extensions/ServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -48,13 +50,32 @@ public static class ServiceCollectionExtensions
...
+            services.Configure<KestrelServerOptions>(options =>
+            {
+                options.Limits.MaxRequestBodySize = maxRequestLimit;
+            });
+
+            services.Configure<FormOptions>(x =>
+            {
+                x.ValueLengthLimit = maxRequestLimit;
+                x.MultipartBodyLengthLimit = maxRequestLimit;
+                x.MultipartHeadersLengthLimit = maxRequestLimit;
```
Comment (gogo4ds) [inline Servers/Infrastructure/OJS.Servers.Infrastructure/Extensions/ServiceCollectionExtensions.cs:73]: I don't think we want the same value for all of these. Either extract different settings for each, or use only what you need.
For example the default value for MultipartBodyLenghtLimit is 128MB (https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.features.formoptions.multipartbodylengthlimit?view=aspnetcore-7.0#microsoft-aspnetcore-http-features-formoptions-multipartbodylengthlimit) but here we make it smaller.
The Headers limit should definitely be smaller than 35MB.

Comment (shsimeonova) [reply Servers/Infrastructure/OJS.Servers.Infrastructure/Extensions/ServiceCollectionExtensions.cs:73]: 🤦🏻‍♀️🤦🏻‍♀️🤦🏻‍♀️

Removed all updates besides `ValueLengthLimit`

File: Services/Common/OJS.Services.Common.Models/Configurations/HttpConfig.cs (hunk 1 above)
Comment (shsimeonova) [reply Services/Common/OJS.Services.Common.Models/Configurations/HttpConfig.cs]: Fixed

File: Servers/Administration/OJS.Servers.Administration/appsettings.json [hunk 3]
```diff
@@ -1,4 +1,7 @@
 {
+  "Http": {
+    "MaxRequestSizeLimit": 35000000
+  },
```
Comment (gogo4ds) [inline Servers/Administration/OJS.Servers.Administration/appsettings.json:4]: Can you put those settings in the bottom, next to AllowedHosts/CORS. Let's keep connection string at the top, as it is changed most frequently.

**Review by gogo4ds — CHANGES_REQUESTED** [review]

File: Servers/Administration/OJS.Servers.Administration/appsettings.json (hunk 3 above)
Comment (shsimeonova) [reply Servers/Administration/OJS.Servers.Administration/appsettings.json]: Fixed

**Review by gogo4ds — APPROVED** [review]

**Pull request merged**